zuul.d/adoption.yaml

```diff
@@ -77,6 +77,74 @@
               tenant:
                 ip: 172.19.0.5
 
+- job:
+    name: cifmw-adoption-standalone-base
+    parent: base-extracted-crc
+    abstract: true
+    timeout: 10800
+    attempts: 1
+    nodeset:
+      nodes:
+        - name: controller
+          label: cloud-centos-9-stream-tripleo-vexxhost-xl
```

Comment:
not a big deal, but we could use `cloud-centos-9-stream-tripleo-vexxhost` here instead, since we no longer have a nested vm and the requirements are much less

Reply:
ack, i will add a TODO(marios) comment here to revisit. having some issues in post-run unregister subscription which is suspect is related to config repo and inheritance. So i want to push to get as much as possible of the series merged and start to iterate on improvements.

Reply:
that's perfectly fine, no need to block on such a detail